packages/app-mobile/components/screens/tags.tsx

```diff
@@ -72,20 +109,75 @@ const TagsScreenComponent: React.FC<Props> = props => {
 		});
 	}, [props.dispatch]);
 
+	const onTagItemLongPress = useCallback(async (tag: TagEntity) => {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Assigning types to these variables would be too big of a refactoring
+		const menuItems: any[] = [];
```

Comment:
```suggestion
		const menuItems: PromptButtonSpec[] = [];
```

It might be possible to use the `PromptButtonSpec` (from [`components/DialogManager/types.ts`](https://github.com/laurent22/joplin/blob/0f4877f263a54057affb232857e131f37cbdccd1/packages/app-mobile/components/DialogManager/types.ts#L10)) type for this.

Reply:
Done